src/main/zapHomeFiles/hud/tools/websockets.js

```diff
@@ -0,0 +1,126 @@
+/*
+ * WebSockets Tool
+ *
+ * Displays websocket messages in a tab at the bottom of the screen.
+ */
+
+var WebSockets = (function() {
+
+	// Constants
+	var NAME = "websockets";
+	var LABEL = I18n.t("websockets_tool");
+	var ICONS = {};
+        ICONS.CLOCK = "clock.png";	// Not actually shown anywhere, should be changed if it is going to be shown
+    var tool = {};
```

Comment:
lots of tab formatting issues, which I see are also in the history tool. We really need to get a linter.

Reply:
feel free to ignore this if you want. would be great to clean up the formatting tho if you can.

Reply:
Have you definitely got the latest code? There were formatting issues before but I think they're fixed now.